judo-tatami-jsl-jsl2ui/src/main/epsilon/operations/jsl/data/transferFieldDeclaration.eol

```diff
@@ -0,0 +1,81 @@
+@cached
+operation JSL!TransferFieldDeclaration getActorDeclaration(): JSL!ActorDeclaration {
+    if (self.eContainer.isKindOf(JSL!TransferDeclaration)) {
+        return self.eContainer.getActorDeclaration();
+    }
+    return null;
+}
+
+@cached
+operation JSL!TransferFieldDeclaration getId(): String {
+    return (self.eContainer.getId() + "/" + self.name);
+}
+
+@cached
+operation JSL!TransferFieldDeclaration isCalculated(): Boolean {
+    return jslUtils.isCalculated(self);
+}
+
+@cached
+operation JSL!TransferFieldDeclaration isRequired(): Boolean {
+    return jslUtils.isRequired(self);
+}
+
+@cached
+operation JSL!TransferFieldDeclaration maps(): Boolean {
+    return jslUtils.isMaps(self);
+}
+
+@cached
+operation JSL!TransferFieldDeclaration reads(): Boolean {
+    return jslUtils.isReads(self);
+}
+
+@cached
+operation JSL!TransferFieldDeclaration getFqName(): String {
+    return self.eContainer.getFqName() + "#" + self.name;
+}
+
+@cached
+operation JSL!TransferFieldDeclaration getTransferFieldDeclarationEquivalent() : UI!ui::data::AttributeType {
+    if (not self.maps() and not self.reads()) {
+        return self.equivalent("CreateTransientTransferAttribute");
+    }
+    if (self.reads()) {
+        return self.equivalent("CreateDerivedTransferAttribute");
+    }
+    if (self.maps()) {
+        return self.equivalent("CreateMappedTransferAttribute");
+    }
+    throw "Could not determinate attribute type: " + s.fqName();
+}
+
+@cached
+operation JSL!TransferFieldDeclaration isSortable() : Boolean {
```

Comment:
I believe enums are sortable as well, hence the name "ordinal"

Reply:
sry, i f****d up the selected code part :D

Reply:
This was copied from jsl2psm, so we'll keep it like this for now. Once we start E2E testing the JSL stack, we can correct it if necessary.